bin/migrate-oats-data/applications/__init__.py

```diff
@@ -1,6 +1,23 @@
 from .base_applications import *
 from .app_prep import process_alcs_application_prep_fields
-from .submissions import process_alcs_app_submissions
+from .submissions import (
+    process_alcs_app_submissions,
```

Comment:
you do not need all these functions here, the applications module should only expose one root function for importing all statuses